crates/store/re_types/src/components/length_ext.rs

```diff
@@ -0,0 +1,10 @@
+use re_types_core::datatypes::Float32;
+
+use super::Length;
+
+impl Default for Length {
+    #[inline]
+    fn default() -> Self {
+        Self(Float32(0.0))
+    }
+}
```

Comment:
You can probably skip this entirely by adding `Default` to the `"attr.rust.derive"` attribute.

Reply:
Zero is a pretty bad default though, I'd rather go with 1.0

Reply:
I think there's some important subtlety here. The bad thing about 0 is that it makes things vanish, but that's a _different kind of default_ than most Rust `Default`s, which are very, very often the “no data here” state, and in that lens, 0 is correct. These are conflicting principles.

That said, for capsules, 0 is well-defined — it makes a sphere.

Reply:
Done. I went with 1.0, but I think this is another case where it would be much better if components weren’t required to have defaults.